ml-agents/mlagents/trainers/ghost/trainer.py

```diff
@@ -101,65 +129,77 @@ def _process_trajectory(self, trajectory: Trajectory) -> None:
             elif final_reward < 0:
                 result = 0.0
 
-            change = compute_elo_rating_changes(
-                self.current_elo, self.policy_elos[self.current_opponent], result
+            change = self.controller.compute_elo_rating_changes(
```

Comment:
Would it make more sense to have compute_elo_rating_changes on the Ghost trainer or is there some data only the controller has access to?

Reply:
The GhostTrainer won't know the ELO ratings of policies associated with another ghost trainer, only its own.  The ghost controller is centralized.